ecllibrary/std/Uni.ecl

```diff
@@ -433,34 +433,34 @@ EXPORT unicode ExcludeLastWord(unicode text, varstring localename = '') :=
  * @return              The string containing the source string but with the translated characters.
  */
 
-EXPORT Translate(unicode text, unicode sear, unicode repl) :=
```

Comment:
Will this break existing code that passes parameters by name?

Reply:
@richardkchapman  Std.Uni.Translate is new and was first made available in 7.0 beta 1. Any ECL code written between beta 1 and 2 that used named params will need to be changed.